app/components/recruiter-conversations/conversation/status.js

```diff
@@ -2,16 +2,15 @@ import Ember from 'ember';
 
 export default Ember.Component.extend({
   profiles: Ember.inject.service(),
-  tagName: 'td',
-  classNames: ['grey'],
-  classNameBindings: ['unreadMessages'],
+  tagName: '',
   unreadMessages: null,
 
 
   onInit: function() {
     const id = this.get('conversation.id');
     return this.get('profiles').getMessages(id)
       .then((bool) => {
+        console.log('unreadmessages:', bool)
```

Comment:
same here, if getMessages(id) returns a boolean lets change the name of it.

You might want to implement a helper method that is something like `.getMessageById` or `.getUnreadMessagesById()` or `getMessageOfConversationById()`